synapseclient/api/configuration_services.py

```diff
@@ -77,18 +77,32 @@ def get_client_authenticated_s3_profile(
     ).get("profile_name", "default")
 
 
-def get_config_authentication(
-    config_path: str,
-) -> Dict[str, str]:
+def get_config_authentication(config_path: str, profile: str = "default"):
```

Comment:
We should keep typing checking for the return types in place

```suggestion
def get_config_authentication(config_path: str, profile: str = "default")  -> Dict[str, str]:
```